server/datastore/psql/create_maintenance_request.go

```diff
@@ -0,0 +1,16 @@
+package psql
```

Comment:
could we move this file to `psql/maintenance_requests.go`? I don't want to have a file per database operation

Reply:
Are you saying I should create a new file that is `package psql` and will contain all future and present database operations? Or move this function to `server/maintenance_requests.go`, which is `package main` and has the API call?

Reply:
new file that's in this folder so it has this package

Reply:
also not _all_ future and present database operations, just the ones pertaining to maintenance requests